eks/cluster/inputs.tf

```diff
@@ -62,4 +62,23 @@ variable "deployer_users" {
   description = "List of AWS User Names create a user in K8S for"
   type        = list(string)
   default     = []
-}
\ No newline at end of file
+}
+
+# Role Info
+variable "system_master_roles" {
+  description = "List of the role name and user to give system master access to"
+  type        = list(object({
+    name      = string
+    user      = string
+  }))
+  default     = []
+}
+
+variable "deployer_roles" {
+  description = "List of the role name and user to give deployer access to"
+  type        = list(object({
+    name      = string
+    user      = string
+  }))
+  default     = []
+}
```

Comment:
Same question on list vs. map here.  It may matter less in this case though, given destroying and recreating may not matter in this case.

Reply:
Yeah for this it shouldn't matter, it would be a very temporary loss of access but it won't recreate any critical resources.

Reply:
And a list is much easier to reason about - with a map you'd have to convert down to a list anyway for the library.